### PR TITLE
Detect Font type Before Convert

### PR DESCRIPTION
**Pull request by greenlikeorange**
Detect font-type before convertion for prevent wrong convertion occur.

### CODE REVIEW

**Comment by setkyar** [comment]
:+1: Nice one, I am trying with drop-down :cry:

**Comment by swanhtet1992** [comment]
@setkyar yours is nice too... :+1: 

@greenlikeorange I will test this after my working hour.

Thank you for awesome contributions guys! 😺

**Comment by swanhtet1992** [comment]
@setkyar @greenlikeorange I tested this but it is not working. Have you guys tested this too?

Actually I love the font detector approach. I want to merge both of you guys commits. :smile:

**Comment by swanhtet1992** [comment]
Meanwhile I will just merge @setkyar pull request #2 first.

@greenlikeorange Could you please fix the issues. So that I could merge.

File: scripts/contentscript.js [hunk 1]
```diff
@@ -19,6 +19,62 @@
...
+  "zaw" : [
+    '\u102c\u1039', '\u103a\u102c', whitespace+'(\u103b|\u1031|[\u107e-\u1084])[\u1000-\u1021]',
+    '^(\u103b|\u1031|[\u107e-\u1084])[\u1000-\u1021]', '[\u1000-\u1021]\u1039[^\u1000-\u1021]', '\u1025\u1039',
+    '\u1039\u1038' ,'[\u102b-\u1030\u1031\u103a\u1038](\u103b|[\u107e-\u1084])[\u1000-\u1021]' ,'\u1036\u102f',
+    '[\u1000-\u1021]\u1039\u1031' , '\u1064','\u1039'+whitespace, '\u102c\u1031',
+    '[\u102b-\u1030\u103a\u1038]\u1031[\u1000-\u1021]', '\u1031\u1031', '\u102f\u102d', '\u1039$'
+  ]
+};
+ 
+function detect(string, default){
```
Comment (swanhtet1992) [inline scripts/contentscript.js:39]: ကိုကြီး `default` က javascript keywoard ကြီးဗျ...

ကိုယ်ကလည်း မလုပ်ဖူးတော့ ဘာမှားမှန်း လျှောက်ရှာလိုက်ရတယ်... :smile: 
screen shot 2015-02-26 at 6 34 30 pm

**Comment by greenlikeorange** [comment]
I created new pull request with new features

**Pull request closed without merging**